src/polar/Pie.tsx

```diff
@@ -305,17 +307,15 @@ function PieSectors(props: PieSectorsProps) {
   });
 }
 
-const getRealPieData = (item: Pie): RealPieData[] => {
-  const { data, children } = item.props;
+const getRealPieData = (item: Pie, cells: ReactElement[]): RealPieData[] => {
```

Comment:
I see that `cells` are being injected later, do we need to pass them as an argument here?

Reply:
Yes so that the cell information can be added to each sector right below this (L322 is hidden, need to expand it)

`cells` are injected later in the selector, but still need to be added to the data for `getComposedData` to continue to work